tasks/ast_tools/src/generators/ast_builder.rs

```diff
@@ -47,8 +47,8 @@ impl Generator for AstBuilderGenerator {
                     clippy::fn_params_excessive_bools,
                 )]
 
-                ///@@line_break
-                use oxc_allocator::{Allocator, Box, IntoIn, Vec};
+                    ///@@line_break
+                    use oxc_allocator::{Allocator, Box, IntoIn, Vec};
```

Comment:
Looks like this extra indentation isn't required.

```suggestion
                ///@@line_break
                use oxc_allocator::{Allocator, Box, IntoIn, Vec};
```